### DISCUSSION TITLE
progressBar specify maximum value

### DISCUSSION THREAD

**Discussion by rastaman111** [LeoNatan/LNPopupController#493] [General] [answered]
Is it possible to specify a maximum value for the progressBar, other than the default 1?  For example, how is the default progressBar?

**Comment by LeoNatan** [comment, marked as answer]
Hello,
You have access to the progress view here:
https://github.com/LeoNatan/LNPopupController/blob/a9696a10488574cc90531d246eb451d694eb84b9/LNPopupController/LNPopupController/LNPopupBar.h#L126

You can configure it as you like.
